### PR TITLE
[ET-1051] Fix override folder comment on templates

### PR DESCRIPTION
**Pull request by andrasguseo**
[ET-1051]
Just fixed some stuff in 2 file headers.

[ET-1051]: https://theeventscalendar.atlassian.net/browse/ET-1051

### CODE REVIEW

File: src/views/tickets/view-link.php [hunk 1]
```diff
@@ -4,7 +4,8 @@
  * The message that will link to the Tickets page.
  *
  * Override this template in your own theme by creating a file at:
- * [your-theme]/tribe/tickets/view-link.php
+ *
+ *     [your-theme]/tribe-events/tickets/tickets/view-link.php
  *
```
Comment (sc0ttkclark) [inline src/views/tickets/view-link.php:8]: Needs an update version and since on any change

**Review by sc0ttkclark — CHANGES_REQUESTED** [review]
Needs updated version/since

File: src/views/tickets/tpp-return-to-cart.php [hunk 2]
```diff
@@ -1,4 +1,18 @@
 <?php
+/**
+ * This template renders the "Return to Cart" text.
+ *
+ * Override this template in your own theme by creating a file at:
+ *
+ *     [your-theme]/tribe-events/tickets/tpp-return-to-cart.php
+ *
+ * @package TribeEventsCalendar
```
Comment (juanfra) [inline src/views/tickets/tpp-return-to-cart.php:9]: @sc0ttkclark what's the name of the package we want to use?

**Comment by rafsuntaskin** [comment]
Screencast:
https://d.pr/v/lxC3sY

**Comment by rafsuntaskin** [comment]
For the `tpp-return-to-cart` template, I was not able to find any usage of that template in the latest version of our plugins, as we don’t have any Cart system for TribeCommerce and the class ( `Tribe__Tickets__Commerce__PayPal__Frontend__Tickets_Form` ) that is rendering that template is not used anywhere any more.

File: src/views/tickets/tpp-return-to-cart.php (hunk 2 above)
Comment (sc0ttkclark) [reply src/views/tickets/tpp-return-to-cart.php:9]: @juanfra we haven't been setting package on these template files, we generally haven't included that and could probably remove the line. Maybe a good question for TEC dev channel to see what standard of usage we want to go with there for template views.

Comment (juanfra) [reply src/views/tickets/tpp-return-to-cart.php:9]: Thanks Scott! @rafsuntaskin I guess removing that line would be the way to go then!

File: src/views/tickets/tpp-return-to-cart.php [hunk 3]
```diff
@@ -1,4 +1,16 @@
 <?php
+/**
+ * This template renders the "Return to Cart" text.
+ *
+ * Override this template in your own theme by creating a file at:
+ *
+ *     [your-theme]/tribe-events/tickets/tpp-return-to-cart.php
```
Comment (juanfra) [inline src/views/tickets/tpp-return-to-cart.php:7]:
```suggestion
 * [your-theme]/tribe-events/tickets/tpp-return-to-cart.php
```

File: src/views/tickets/view-link.php [hunk 4]
```diff
@@ -4,7 +4,12 @@
  * The message that will link to the Tickets page.
  *
  * Override this template in your own theme by creating a file at:
- * [your-theme]/tribe/tickets/view-link.php
+ *
+ *     [your-theme]/tribe-events/tickets/view-link.php
+ *
+ * If you are using ETP and V2 templates ( new form experience ) then create a file at:
+ *
+ *     [your-theme]/tribe/tickets/tickets/view-link.php
```
Comment (juanfra) [inline src/views/tickets/view-link.php:12]:
```suggestion
 * [your-theme]/tribe-events/tickets/view-link.php
 *
 * If you are using Event Tickets Plus and V2 templates (new form experience) then create a file at:
 *
 * [your-theme]/tribe/tickets/tickets/view-link.php
```

**Review by juanfra — COMMENTED** [review]
Looking good! We need to add a changelog

File: readme.txt [hunk 5]
```diff
@@ -183,6 +183,7 @@ Check out our extensive [knowledgebase](https://evnt.is/18wm) for articles on us
 * Fix - Fixed an issue with Tickets and RSVP blocks where long descriptions were breaking the block. They now use an auto-resizing textarea. [ET-1078]
 * Tweak - Introduce a new "Attendees" link to the WP Admin bar which can take you directly to the Attendees Report page. [ET-1079]
 * Tweak - Added the new `tribe_tickets_attendees_csv_export_delimiter` filter to allow changing the delimiter used when generating a CSV export of attendees. [ET-1055]
+* Tweak - Adjusted template override folder path in template's header comment. [ET-1051]
```
Comment (sc0ttkclark) [inline readme.txt:186]:
```suggestion
* Tweak - Adjusted some template override folder paths documented in some of our Tickets-related templates. [ET-1051]
```

**Review by sc0ttkclark — APPROVED** [review]

**Pull request merged**